src/ecl_ekf_analysis/config/thresholds.ini

```
@@ -67,8 +67,8 @@ imu_high_freq_delta_velocity_warning_rolling_avg = 0.2
 imu_observed_angle_error_warning_avg = 8.0E-3
 imu_observed_velocity_error_warning_avg = 0.05
 imu_observed_position_error_warning_avg = 0.15
-imu_delta_angle_bias_warning_avg = 4.4E-4
-imu_delta_velocity_bias_warning_avg = 4.8E-3
+imu_delta_angle_bias_warning_avg = 4.4E-2
```

```suggestion
imu_gyro_bias_warning_avg = 4.4E-2
```
Could we maybe also rename those? (delta_vel -> accel as well)

---

We could change it here, but updating the gRPC message names is not currently possible - meaning there will be a naming mismatch between messages when they are read by Suite. A complete update will be possible once we have the Rust services compiled again. Is it still accurate to keep the variables named as is for consistency, or does it make more sense to update as much as we can to use gyro/accel? Apologies for the inconvenience here, we're working on it!

---

It's fine, we can do it later. The most important part was the user-facing names.